### PR TITLE
Dashboard for Log Analysis

### PR DESCRIPTION
**Pull request by thcanate**
# Summary

A concise description of the changes being introduced. If you are submitting a PR for a
new quickstart please review the pre merge checklist section to validate this PR is ready for review.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [x] Did you check that your alerts actually work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.
log-analysis

### CODE REVIEW

**Review by roadlittledawn — CHANGES_REQUESTED** [review]

File: importer-config.sh [hunk 1]
```diff
@@ -3,4 +3,4 @@
 # How to find your accountId: https://docs.newrelic.com/docs/accounts/accounts-billing/account-setup/account-id/
 accountId=
 # How to create a User API Key: https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#user-key-create
-apiKey=
+apiKey=
```
[the remark this answers is not in the document]
Comment (thcanate) [reply importer-config.sh:6]: Removed the file from gitignore

File: quickstarts/log-analysis/dashboards/log-analysis.json [hunk 2]
```diff
@@ -0,0 +1,403 @@
+{
+  "name": "Logs Analysis",
+  "description": null,
+  "permissions": "PRIVATE",
```
[the remark this answers is not in the document]
Comment (thcanate) [reply quickstarts/log-analysis/dashboards/log-analysis.json]: Removed the permission

File: quickstarts/log-analysis/config.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  - data
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - log-docs
+
+documentation:
+  - name: Log Management
+    url: https://docs.newrelic.com/docs/logs/
```
[the remark this answers is not in the document]
Comment (thcanate) [reply quickstarts/log-analysis/config.yml]: changed the URL link.

**Review by roadlittledawn — APPROVED** [review]
Thanks for making those changes! LGTM. :shipit:

**Pull request merged**